src/app/shared/guide-items/guide-items.ts

```diff
@@ -4,58 +4,78 @@ export interface GuideItem {
   id: string;
   name: string;
   document: string;
+  overview: string;
 }
 
 const GUIDES = [
   {
     id: 'getting-started',
     name: 'Getting started',
     document: '/docs-content/guides/getting-started.html',
+    overview: 'Setup your Angular project to begin using Angular Material. Learn about' +
+      ' prerequisites, installing Angular Material, and optionally displaying a sample material' +
+      ' component in your application to verify your setup.'
   },
   {
     id: 'schematics',
     name: 'Schematics',
-    document: '/docs-content/guides/schematics.html'
+    document: '/docs-content/guides/schematics.html',
+    overview: 'Angular Material comes packaged with Angular CLI schematics to make creating ' +
+      'Material applications easier.'
   },
   {
     id: 'theming',
     name: 'Theming Angular Material',
     document: '/docs-content/guides/theming.html',
+    overview: 'Define your own themes through Angular Material\'s tooling with Sass.'
```

Comment:
Maybe
```
Customize your application with Angular Material's theming system.
```
Also you can use `"` or `` ` `` for the string to avoid escaping the `'` (here and elsehwere)